README.md

```diff
@@ -243,6 +244,30 @@ See gif in action:
 The built-in multi-cursor support may not meet your needs. Please refer to the plugin
 [vscode-multi-cursor.nvim](https://github.com/vscode-neovim/vscode-multi-cursor.nvim) for more multi-cursor features
 
+### Remote development
+
+We intend to use vscode-neovim as a UI extension, so when you're using remote development, vscode-neovim is enabled in
+the Local Extension Host, and it should work out of the box.
+
+If you prefer to use Neovim in the remote environment, vscode-neovim should be installed in the Remote Extension Host.
```

Comment:
```suggestion
If you prefer to use the remote environment's copy of Neovim, rather than the locally installed one, vscode-neovim should be installed in the Remote Extension Host.
```

Better, just a small tweak you can take or leave